### ISSUE TITLE
NVDA does not read the control type of a role="menuitem" in button element

### ISSUE THREAD

**Issue by satbai** [nvaccess/nvda#8654]
### Steps to reproduce:
NVDA does not read the role "menuitem" on a button element.

Tab through the button elements and check NVDA speech viewer. 
```
<div role="menu">
      <button role="menuitem">two</button> 
      <button role="menuitem">three</button> 
</div>
```

### Actual behavior:
NVDA reads: 
two 1 of 2
three 2 of 2

![image](https://user-images.githubusercontent.com/24945574/44359113-c0543480-a46b-11e8-825b-7d18db889c4f.png)

### Expected behavior:
NVDA should read the control type, something like:
two, menu item, 1 of 2
three, menu item, 2 of 2

This is what for example Windows Narrator reads.
![image](https://user-images.githubusercontent.com/24945574/44360085-89335280-a46e-11e8-8bd7-03457d03ea94.png)

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version: 
nvda_2018.2.1

#### Windows version: 
Windows 10 version 1803

#### Name and version of other software in use when reproducing the issue:
Chrome Version 67.0.3396.99
Microsoft Edge 42.17134.1.0

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
This is also reproducible in Firefox 66.0.4.
But note that this occurs only in focus mode when navigating with tab and shift+tab. When using browse mode, NVDA reads the controltype.
@jcsteh is this intended behavior, at least for Firefox?

**Comment by jcsteh** [comment]
This is intentional. Hearing the menuitem role doesn't provide useful information to the user. If the user is in a menu, they have enough context to know that they're dealing with menu items. Hearing "menu item" every time they move to an option is annoying, increases cognitive load and decreases efficiency.
